README.md

```diff
@@ -59,6 +59,14 @@ This project has been worked on by many Lambda labs teams over the past 10 month
 
 The BERT model does not currently live in the GitHub repository due to its large file size. When running the app locally, it is best to manually store the `saved_model` file in the `app` directory.
 
+## BERT rankings
+Taking a deeper dive we can turn our eyes to the black box of our model. For this task we will use LIME. LIME is an acronym for local interpretable model-agnostic explanations. Local is refers to local fidelity, meaning we want the explanation to really reflect the behaviour of the classifier "around" the instance being predicted. Interpretable refers to making sense of these explanations. Lastly,  model-agnostic refers to giving explanations without needing to ‘peak’ into it.  
```

Comment:
On line 32, I found a typo: 'Local is refers to local fidelity,...'

This feels as though the word 'is' was meant to be deleted. Not a merge blocker, just a nitpick for future change!